### PR TITLE
 Improve messages for rrdcached daemon failed due to PID Exist

### PR DESCRIPTION
**Pull request by Manoj-red-hat**
New messages added in order to get clarity about the nature of problem

### CODE REVIEW

**Comment by Manoj-red-hat** [comment]
Thanks a lot @oetiker Sir,
For teaching me how to contribute in OSS  :)

File: src/rrd_daemon.c [hunk 1]
```diff
@@ -524,16 +524,24 @@ static int check_pidfile(void)
   char pid_str[16];
 
   pid_fd = open_pidfile("open", O_RDWR);
-  if (pid_fd < 0)
+  if (pid_fd < 0){
+     fprintf(stderr,
+                    "FATAL: Fail to create/open PID file");
     return pid_fd;
+     }
```
Comment (oetiker) [inline src/rrd_daemon.c]: there seems to be an indentation problem

Comment (oetiker) [inline src/rrd_daemon.c]: there seems to be an indentation problem

File: src/rrd_daemon.c [hunk 2]
```diff
@@ -524,16 +524,24 @@ static int check_pidfile(void)
   char pid_str[16];
 
   pid_fd = open_pidfile("open", O_RDWR);
-  if (pid_fd < 0)
+  if (pid_fd < 0){
+     fprintf(stderr,
+                    "FATAL: Fail to create/open PID file");
```
Comment (oetiker) [inline src/rrd_daemon.c]: please keep fprint on one line

File: src/rrd_daemon.c [hunk 3]
```diff
@@ -524,16 +524,24 @@ static int check_pidfile(void)
...
     return pid_fd;
+     }
 
   if (read(pid_fd, pid_str, sizeof(pid_str)) <= 0) {
+     fprintf(stderr,
+                     "FATAL: Empty PID file exist");
```
Comment (oetiker) [inline src/rrd_daemon.c]: please keep fprint on one line

File: src/rrd_daemon.c [hunk 4]
```diff
@@ -524,16 +524,24 @@ static int check_pidfile(void)
...
     close(pid_fd);
     return -1;
   }
 
   pid = atoi(pid_str);
   if (pid <= 0) {
+     fprintf(stderr,
+                     "FATAL: PID file is corrupted");
```
Comment (oetiker) [inline src/rrd_daemon.c]: please keep fprint on one line

**Comment by oetiker** [comment]
thanks for your patch

**Pull request merged**